### PR TITLE
Modify homepage hero to full width (#698)

### PR DESCRIPTION
**Pull request by saranglove**
Just the jumbotron.

Things to continue working on:
Little details on the navbar
The cards and links

### CODE REVIEW

File: csunplugged/static/scss/website.scss [hunk 1]
```diff
@@ -101,16 +100,21 @@ div.plugged-in-language-implementation {
...
 
 .jumbotron {
-  background: linear-gradient( rgba($red,.7), rgba($red,.7) ), url("../img/home-page-banner-photo.jpg");
+  background: url("../img/red-hero-banner.png");
   background-size: cover;
-  border: $card-border-width solid darken($red, 10%);
-  border-radius: $rounded-corner-radius;
-  margin: 1rem 0;
   width: 100%;
+  padding-bottom: 15vh;
```
Comment (JackMorganNZ) [inline csunplugged/static/scss/website.scss]: Is there a reason for this much padding?

File: csunplugged/templates/general/index.html [hunk 2]
```diff
@@ -7,61 +7,62 @@
...
-          {% endblocktrans %}
-        </h1>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 class="display-3 text-center">
+      {% blocktrans trimmed %}
+      Computer Science<br>without a computer
+      {% endblocktrans %}
+    </h1>
+    <div class="container">
```
Comment (JackMorganNZ) [inline csunplugged/templates/general/index.html]: The `container` within the `container` is not required.

File: csunplugged/templates/general/index.html [hunk 3]
```diff
@@ -7,61 +7,62 @@
...
+      <div class="row align-items-center">
+        <div class="col-md-1 d-none d-md-block">
```
Comment (JackMorganNZ) [inline csunplugged/templates/general/index.html]: This element is empty and should be removed.

File: csunplugged/templates/general/index.html [hunk 4]
```diff
@@ -7,61 +7,62 @@
...
+            CS Unplugged is a collection of free teaching material that teaches Computer Science through engaging games and puzzles that use cards, string, crayons and lots of running around.
+            {% endblocktrans %}
+          </p>
+          <p class="lead text-center">
+            {% blocktrans trimmed %}
+            The resources available to teachers include unit plans, lesson plans, teaching videos, curriculum integration activities, and programming exercises to plug in the Computer Science concepts they have just learnt unplugged.
+            {% endblocktrans %}
+          </p>
+        </div>
+        <div class="col-md-1 d-none d-md-block">
```
Comment (JackMorganNZ) [inline csunplugged/templates/general/index.html]: This element is empty and should be removed.

File: csunplugged/templates/general/index.html [hunk 5]
```diff
@@ -7,61 +7,62 @@
...
-    <div class="jumbotron jumbotron-fluid py-5 mt-0">
-      <div class="container">
-        <h1 class="display-3 text-center">
-          {% blocktrans trimmed %}
-          Computer Science<br>without a computer
-          {% endblocktrans %}
-        </h1>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 class="display-3 text-center">
```
Comment (JackMorganNZ) [inline csunplugged/templates/general/index.html]: The `display-3` class is unused so can be removed.

File: csunplugged/templates/general/index.html [hunk 6]
```diff
@@ -7,61 +7,62 @@
...
+      {% blocktrans trimmed %}
+      Computer Science<br>without a computer
+      {% endblocktrans %}
+    </h1>
+    <div class="container">
+      <div class="row align-items-center">
```
Comment (JackMorganNZ) [inline csunplugged/templates/general/index.html]: `align-items-center` doesn't seem to be needed here.

File: csunplugged/templates/general/index.html [hunk 7]
```diff
@@ -7,61 +7,62 @@
...
-  <div class="row">
-    <div class="jumbotron jumbotron-fluid py-5 mt-0">
-      <div class="container">
-        <h1 class="display-3 text-center">
-          {% blocktrans trimmed %}
-          Computer Science<br>without a computer
-          {% endblocktrans %}
-        </h1>
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
```
Comment (JackMorganNZ) [inline csunplugged/templates/general/index.html]: I would add `text-center` here.

**Comment by JackMorganNZ** [comment]
This is relevant to #698.

File: csunplugged/static/scss/website.scss (hunk 1 above)
Comment (saranglove) [reply csunplugged/static/scss/website.scss]: I have it set ready for the cards to overlap the jumbotron.
image

Comment (JackMorganNZ) [reply csunplugged/static/scss/website.scss]: Excellent.

**Pull request merged**